src/app/public/modules/action-button/action-button-icon.component.ts

```diff
@@ -16,15 +16,26 @@ import {
 const FONTSIZECLASS_SMALL = '2x';
 const FONTSIZECLASS_LARGE = '3x';
 
+/**
+ * Specifies an icon to display on the action button.
+ */
 @Component({
   selector: 'sky-action-button-icon',
   styleUrls: ['./action-button-icon.component.scss'],
   templateUrl: './action-button-icon.component.html'
 })
 export class SkyActionButtonIconComponent implements OnDestroy {
 
+/**
+ * Specifies an icon from the
+ * [Font Awesome library](https://fontawesome.com/v4.7.0/).
+ * For example, to display the `fa-filter` icon on the action button,
+ * set `iconType` to `filter`. SKY UX supports version 4.7.0 of Font Awesome.
+ * For more information about icons in SKY UX, see the
+ * [icons design pattern](https://developer.blackbaud.com/skyux/design/styles/icons).
```

Comment:
I think `icons design pattern` should be `icon component` now that the icon design pattern content was moved to the component. We could also update the link here, but we have a redirect in place to handle that automatically.